assets/dev/js/editor/regions/panel/pages/elements/views/elements.js

```diff
@@ -22,12 +22,16 @@ PanelElementsElementsView = Marionette.CollectionView.extend( {
 			return false;
 		}
 
-		// Remove widgets from search results by 'widgets categories'.
-		const showInSearchResult = Object.keys( elementor.documents.getCurrent().config.panel.elements_categories )
-			.some( ( categoryName ) => childModel.get( 'categories' ).includes( categoryName ) );
-
-		if ( ! showInSearchResult ) {
-			return false;
+		const isGlobalWidget = childModel.get( 'custom' )?.templateID;
```

Comment:
Please add the global widget category into the list and if there is no category make sure that widget without category will be shown

Reply:
We decided to skit this since there is no magic solution